examples/medplum-patient-intake-demo/data/core/patient-intake-questionnaire.json

```diff
@@ -85,6 +85,36 @@
                 "text": "Date of Birth",
                 "type": "date"
               },
+              {
+                "linkId": "street",
+                "text": "Street",
+                "type": "string"
+              },
+              {
+                "linkId": "city",
+                "text": "City",
+                "type": "string"
+              },
+              {
+                "linkId": "state",
+                "text": "State",
+                "type": "string"
```

Comment:
Do we want to make this a ValueSet?  Seems like it would have repeated utility

Reply:
I will update the code to use this US Core ValueSet: http://hl7.org/fhir/us/core/ValueSet/us-core-usps-state

Reply:
I found that in the [US Core Location Profile](http://hl7.org/fhir/us/core/StructureDefinition/us-core-location)